tpl-implementations/include/experimental/__p1673_bits/kokkos-kernels/blas1_scale_kk.hpp

```diff
@@ -3,17 +3,35 @@
 #define LINALG_TPLIMPLEMENTATIONS_INCLUDE_EXPERIMENTAL___P1673_BITS_KOKKOSKERNELS_SCALE_HPP_
 
 namespace KokkosKernelsSTD {
-template<class ExecSpace,
+
+//
+// for now, specialize for default_accessor
+// https://github.com/kokkos/stdBLAS/issues/122
+//
+template<class ExeSpace,
          class Scalar,
          class ElementType,
          std::experimental::extents<>::size_type ... ext,
-         class Layout,
-         class Accessor>
-void scale(kokkos_exec<ExecSpace>, const Scalar alpha,
-           std::experimental::mdspan<ElementType, std::experimental::extents<ext ...>, Layout, Accessor> x)
+         class Layout>
+void scale(kokkos_exec<ExeSpace> /*kexe*/,
+       const Scalar alpha,
+           std::experimental::mdspan<
+         ElementType,
+         std::experimental::extents<ext ...>,
+         Layout,
+         std::experimental::default_accessor<ElementType>
+       > obj)
 {
-    KokkosBlas::scal(Impl::mdspan_to_view(x),alpha,Impl::mdspan_to_view(x));
-}
+  // constraints
+  static_assert(obj.rank() <= 2);
+
+#if defined KOKKOS_STDBLAS_ENABLE_TESTS
+  std::cout << "scale: kokkos impl\n";
```

Comment:
Please see my comment on e.g., PR #160 about encapsulating logging.  This is not required but would be a good idea (I think : - ) ).  Thanks!